lib/streamlit/error_util.py

```diff
@@ -42,15 +44,33 @@ def handle_uncaught_app_exception(e: BaseException) -> None:
     if the user has disabled client error details, we display a generic
     warning in the frontend instead.
     """
+    exc_type, exc_value, exc_traceback = sys.exc_info()
     if config.get_option("client.showErrorDetails"):
         LOGGER.warning(traceback.format_exc())
+        st.warning(
+            "Having showErrorDetails to be true can lead to secrets being leaked!"
```

Comment:
nit: "to be true" -> "set to true"? (product is likely going to want to wordsmith the messages here anyway)